packages/orm/src/EntityManager.ts

```diff
@@ -1065,6 +1083,13 @@ export class EntityManager<C = unknown> {
 
     this._isFlushing = true;
 
+    const touched = this.entities.filter((e) => e.__orm.isTouched);
```

Comment:
`em.touch` used to go through `Todo.asyncFields`, but since that went away to decouple reactivity from the em.flush loops, we just handle touching here explicitly.